README.rst

```diff
@@ -260,11 +260,3 @@ make install
 
 The default installation prefix is /usr/local.  You may need to run
 'ldconfig' after performing a 'make install'.
-
```

Comment:
XenAccess is far enough in the past that I agree it is ok to remove it from here.  I don't think we need a history section... we can just remove it.